manifests/katello.pp

```diff
@@ -46,5 +54,35 @@
     bootstrap_script => template('certs/rhsm-katello-reconfigure.erb'),
     alias            => $candlepin_cert_rpm_alias,
     subscribe        => $::certs::server_ca,
+  } ~>
```

Comment:
This still ties the atomic workflow to the certs_bootstrap_rpm, I think it would be better to break this dependency and let the atomic stuff stand alone i.e. change this line to `}` and insert a blank line.